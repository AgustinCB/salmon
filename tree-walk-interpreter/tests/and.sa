print "HOLA" and "CHAU";
print nil and "HOLA";
print "HOLA" and false;
print false and "HOLA";
print nil and nil;