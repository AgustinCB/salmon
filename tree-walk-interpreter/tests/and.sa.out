CHAU
Nil
false
false
Nil
